packages/harpguru-core/src/styles/styles-types.ts

```diff
@@ -42,3 +42,8 @@ export type SizeScheme = {
   readonly labelProtrusion: number
   readonly labelIconSize: number
 }
+
```

Comment:
I'd like to split these apart so that the static sizes are only updated when the screen dimensions change. The dynamic sizes are liable to update more frequently.

Reply:
Raised as techdebt here: https://github.com/js-jslog/harpguru/issues/113#issuecomment-922036125

The work done to make the app sizing dynamic (at #107) was used as a platform for additional speculative pull requests to ensure that this was the correct direction to start down.

Rather than addressing the finer details in this or subsequent pull request and potentially struggling to merge up to those subsequent branches I will prefer to record the issues in each merge request and resolve them afterwards.

This has a limit of course. Major issues will not be merged to master. I will need to find a way around that if it happens.